internal/mesh/exports.go

```diff
@@ -50,10 +56,36 @@ var (
 	TCPRouteType                        = types.TCPRouteType
 	DestinationPolicyType               = types.DestinationPolicyType
 	ComputedRoutesType                  = types.ComputedRoutesType
+
+	// Controller statuses.
+
+	// Sidecar-proxy controller.
```

Comment:
These comments don't seem to match the surroundings ideally.

Reply:
How so? Are you referring to spacing?